kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2087,20 +2087,33 @@ object KyuubiConf {
       .version("1.5.0")
       .fallbackConf(ENGINE_CONNECTION_URL_USE_HOSTNAME)
 
+  val ENGINE_DO_AS_ENABLED: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.doAs.enabled")
```

Comment:
cc @yaooqinn @zhouyifan279 WDYT of this configuration name? My another candidate is `kyuubi.engine.impersonation.enabled`

Reply:
+1 for kyuubi.engine.doAs.enabled.